File: MiniDOMAsteriskLib/CallManagers/Events/UserEvent.vb
Imports System.Net.Sockets
Imports System.Threading
Imports System.Net

Namespace CallManagers.Events

    Public Class UserEvent
        Inherits AsteriskEvent
         

        Public Sub New()
            MyBase.New("UserEvent")
        End Sub

        Public Sub New(ByVal rows() As String)
            MyBase.New(rows)
        End Sub

        Public ReadOnly Property Channel As String
            Get
                Return Me.GetAttribute("Channel")
            End Get
        End Property

        Public ReadOnly Property Uniqueid As String
            Get
                Return Me.GetAttribute("Uniqueid")
            End Get
        End Property
         
    End Class

End Namespace

File: MiniDOMAsteriskLib/CallManagers/Events/UserEvent.cs

namespace minidom.CallManagers.Events
{
    public class UserEvent : AsteriskEvent
    {
        public UserEvent() : base("UserEvent")
        {
        }

        public UserEvent(string[] rows) : base(rows)
        {
        }

        public string Channel
        {
            get
            {
                return GetAttribute("Channel");
            }
        }

        public string Uniqueid
        {
            get
            {
                return GetAttribute("Uniqueid");
            }
        }
    }
}